ballerina/tests/toXml_test.bal

```diff
@@ -1330,6 +1330,41 @@ isolated function testToRecordFieldNameEndsWithNameOrNamespace() returns error?
     test:assertEquals(result.toString(), expected);
 }
 
+type VersionType string;
```

Comment:
Shall we have another reference for this this type.

Reply:
Updated https://github.com/ballerina-platform/module-ballerina-data.xmldata/pull/32/commits/28079317a20e8b5abca9b8fb71a609c70ad1c580